sources/packages/backend/apps/api/src/app.aest.module.ts

```diff
@@ -134,6 +141,7 @@ import { WorkflowClientService, SequenceControlService } from "@sims/services";
     EducationProgramOfferingControllerService,
     EducationProgramOfferingValidationService,
     DisbursementReceiptService,
+    QueueService,
```

Comment:
`QueueModule` is declared as `@Global` and also imported on `app.module.ts`. is it not the case of exporting the `QueueService` from the `QueueModule`?